src/components/LazyLog/index.tsx

```diff
@@ -1180,65 +1194,19 @@ export default class LazyLog extends Component<LazyLogProps, LazyLogState> {
                         iconFindPrevious={this.props.iconFindPrevious}
                     />
                 )}
-
-                {/*
-                 // @ts-ignore */}
-                <AutoSizer
-                    disableHeight={this.props.height !== "auto"}
-                    disableWidth={this.props.width !== "auto"}
+                <VList
+                    className={`react-lazylog ${styles.lazyLog} ${
+                        this.props.wrapLines ? styles.wrap : ""
+                    }`}
+                    style={{ height: 800 }}
```

Comment:
Hardcoded 800? Instead of calculateListHeight?

Reply:
Yeah, that was just for getting the POC up and running. Now that I know there's interest Ill spend some time cleaning it up.